packages/plugins/integration-zapier/zapier/src/creates/stopTimer.ts

```
@@ -25,9 +25,10 @@ const perform = async (z: ZObject, bundle: Bundle) => {
       },
     });
     return response.data;
-  } catch (error) {
-    throw new Error('Failed to stop timer');
-  }
+  } catch (error: any) {
+    z.console.error('Error stopping timer:', { error });
+    throw new Error(`Failed to stop timer: ${error.message || 'Unknown error'}`);
+  };
```

Biome (1.9.4)

[error] This code is unreachable

... because either this statement will return from the function ...

... or this statement will throw an exception beforehand


(lint/correctness/noUnreachable)